3.analysis/scripts/nbconverted/2.gene_set_enrichment_analysis.py

```diff
@@ -114,38 +133,76 @@
 plt.title('Control Gene Set Enrichment Analysis')
 
 
-# In[10]:
+# In[11]:
 
 
 # Using VAE generated data
 
+pdf_path = "../1.data-exploration/figures/gsea_plots.pdf"
```

Comment:
```suggestion
pdf_path = pathlib.Path("../1.data-exploration/figures/gsea_plots.pdf")
```
Seems like you're creating a path here. 

Please consider using [`pathlib`](https://docs.python.org/3/library/pathlib.html) to manage paths instead of strings. This will help ensure reproducibility and prevent path based errors. 

If there is a reason why you're using strings types for pathing, feel free to ignore this suggestion.

Reply:
Will do! Thanks for the suggestion